setup.py

```diff
@@ -1,53 +1,64 @@
 """Package Setup script for TFX Addons."""
+import datetime
 import itertools
 import os
-import re
+import sys
 
 from setuptools import find_namespace_packages, setup
 
 
-def _get_version():
-  version_file = os.path.join(os.path.dirname(__file__),
-                              'tfx_addons/__init__.py')
-  with open(version_file, 'r') as fp:
-    version_file_text = fp.read()
+def get_last_commit_time() -> str:
+    string_time = os.getenv("NIGHTLY_TIME").replace('"', "")
+    return datetime.strptime(string_time, "%Y-%m-%dT%H:%M:%SZ").strftime("%Y%m%d%H%M%S")
 
-  version_match = re.search(
-      r"^__version__ = ['\"]([^'\"]*)['\"]",
-      version_file_text,
-      re.M,
-  )
-  if version_match:
-    return version_match.group(1)
-  else:
-    raise RuntimeError("Unable to find version string.")
+
+def get_project_name_version():
+    # Version
+    version = {}
+    base_dir = os.path.dirname(os.path.abspath(__file__))
+    with open(os.path.join(base_dir, "tfx_addons", "version.py")) as fp:
+        exec(fp.read(), version)
+
+    project_name = "tfx-addons"
+    # TODO: Support nightly builds? Like https://github.com/tensorflow/addons/blob/d19c3efa5e4b9924e5f1e0b603fe0a22b8efcbfe/.github/workflows/release.yml#L65
+    if "--nightly" in sys.argv:
+        project_name = "tfx-addons-nightly"  # TODO: Confirm project names with rcrowe
```

Comment:
do we really think we will use nightly releases?

Reply:
removed